README.md

```diff
@@ -15,6 +15,25 @@ This version will look for an `app_config.yml` file on `config` folder in the pr
 - Add option for server config, right now only creates nginx vhost file and mysql database files for rails AR projects.
 - More stuff
 
+# Use
+In a rails app, simply include the gem in your Gemfile:
+```
+gem 'biran'
+```
+
+In a non-rails app, you will need to do some extra work to make the tasks available to rake. In your Rakefile you will need to manually include things.
+Here is a minimal Rakefile for a basic ruby app that includes the biran tasks along with any local tasks in `lib/tasks`
+```
+require 'bundler/setup'
+require 'biran'
```

Comment:
I still think using `Bundler.require` is a best advise. Still is on the domain of each ruby project to decide how to load dependencies, but is very standard to do `Bundler.require` and is what `Bundler` recommends

Reply:
Sorry... just copied and pasted from earlier work prior to the previous conversation.
So should that be something like
```
require bundler/setup
Bundler.require
```